examples/custom-vnet/README.md

```diff
@@ -84,7 +84,7 @@ resource "azurerm_virtual_network" "example" {
 module "lacework_azure_agentless_scanning_rg_and_vnet" {
   source = "lacework/agentless-scanning/azure"
 
-  integration_level              = "SUBSCRIPTION"
+  integration_level              = "TENANT"
```

Comment:
I changed this example to tenant-level integration to avoid having to specify `included_subscriptions`